docker/pre_push_hook.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For this script to work properly, you need to have scripts/pre_push_hook.py
+# This scripts helps running pre-push hook inside docker container by,
+# working as middle layer beteen hook and pre_push_hook.py in docker
+# If you can't understand whats going on here, please visit
+# https://docs.google.com/document/d/1whHSUEchpmnqxSsRvHU_WYbBzeqVMh7vrjpuA8FMnAM/edit?usp=sharing
+#
+# To install hook (which runs inside docker container), run this script
+# command from oppia root directory with --install argument
+#
+# Currently it only works for Unix systems, on windows it will create symlink but won't have any effect
+
+DEV_CONTAINER="dev-server"
+DOCKER_EXEC_COMMAND="docker compose exec -T $DEV_CONTAINER "
+
+
+# Location of git hooks directory
+HOOKS_DIR=".git/hooks"
+
+# Path for pre-push hook file
```

Comment:
Rephrases the comments to mention it is the symlink

Reply:
done!